windows-observ-lib/dashboards.libsonnet

```diff
@@ -0,0 +1,156 @@
+              panels.cpuUsageTopk { gridPos+: { w: 24 } },
+              panels.memotyUsageTopKPercent { gridPos+: { w: 24 } },
+              panels.diskIOutilPercentTopK { gridPos+: { w: 12 } },
+              panels.diskUsagePercentTopK { gridPos+: { w: 12 } },
+              panels.networkErrorsAndDroppedPerSec { gridPos+: { w: 24 } },
+            ], 12, 7
+          )
+        )
+        // hide link to self
+        + root.applyCommon(vars.multiInstance, uid + '-fleet', tags, links { backToFleet+:: {}, backToOverview+:: {} }, annotations, timezone, refresh, period),
+      overview: g.dashboard.new(prefix + 'Windows overview')
+                + g.dashboard.withUid('overview')
+                + g.dashboard.withPanels(
+                  g.util.grid.wrapPanels(
+                    [
+                      g.panel.row.new('Overview'),
+                      panels.uptime,
+                      panels.hostname,
+                      panels.osVersion,
+                      panels.osInfo,
+                      panels.cpuCount,
+                      panels.memoryTotalBytes,
+                      panels.memoryPageTotalBytes,
+                      panels.diskTotalC,
+                      g.panel.row.new('CPU'),
+                      panels.cpuUsageStat { gridPos+: { w: 6, h: 6 } },
+                      panels.cpuUsageTs { gridPos+: { w: 18, h: 6 } },
+                      g.panel.row.new('Memory'),
+                      panels.memoryUsageStatPercent { gridPos+: { w: 6, h: 6 } },
+                      panels.memoryUsageTsBytes { gridPos+: { w: 18, h: 6 } },
+                      g.panel.row.new('Disk'),
+                      panels.diskIOBytesPerSec { gridPos+: { w: 12, h: 8 } },
+                      panels.diskUsage { gridPos+: { w: 12, h: 8 } },
+                      g.panel.row.new('Network'),
+                      panels.networkUsagePerSec { gridPos+: { w: 12, h: 8 } },
+                      panels.networkErrorsPerSec { gridPos+: { w: 12, h: 8 } },
+                    ], 6, 2
+                  )
+                )
+                + root.applyCommon(vars.singleInstance, uid + '-overview', tags, links { backToOverview+:: {} }, annotations, timezone, refresh, period),
+      // add TODO advanced memory dashboard (must enable memory collector)
+      // memory:
+
+      system: g.dashboard.new(prefix + 'Windows CPU and system')
+              + g.dashboard.withPanels(
+                g.util.grid.wrapPanels(
+                  [
+                    g.panel.row.new('System'),
+                    panels.cpuUsageStat { gridPos+: { w: 6, h: 6 } },
+                    panels.cpuUsageTs { gridPos+: { w: 9, h: 6 } },
+                    panels.cpuUsageByMode { gridPos+: { w: 9, h: 6 } },
+                    panels.cpuQueue,
+                    panels.systemContextSwitchesAndInterrupts,
+                    // panels.systemThreads,
+                    // panels.systemExceptions,
```

Comment:
Is there a problem with these? Should they also be removed from the windows-observ-lib?

Reply:
I'll check and make follow-up PR.